docs/src/doc/dev/adr/0002-wikimedia-ui-base-variable-reuse.stories.mdx

```diff
@@ -1,3 +1,7 @@
+import { Meta } from '@storybook/addon-docs/blocks';
```

Comment:
I'm not sure how I feel about this move myself yet. It certainly gives them visibility and shows that consideration indeed has been put into those decisions.